packages/client/src/addresses/getAddressSchema.ts

```diff
@@ -0,0 +1,22 @@
+import { adaptError } from '../helpers/client/formatError.js';
+import client from '../helpers/client/index.js';
+import join from 'proper-url-join';
+import type { GetAddressSchema } from './types/index.js';
+
+/**
+ * Method responsible for getting the predictions.
+ *
+ * @param countryCode   - country code to get the schema.
+ * @param config - Custom configurations to send to the client instance (axios).
+ *
+ * @returns Promise that will resolve when the call to the endpoint finishes.
+ */
+const getAddressSchema: GetAddressSchema = (countryCode, config) =>
```

Comment:
This client is already implemented as `getCountryAddressSchemas`. You can remove it.

Reply:
done